paper/basic_training.tex

```diff
@@ -731,8 +787,28 @@ \subsection{Integrators}
 \item Numerical solution of dynamical equations of motions
 \item Importance of energy conservation
 \item Commonly used integrators
-\item How to choose an appropriate timestep?
 \end{itemize}
+\subsubsection{How to choose an appropriate timestep?} The maximum
+timestep for a molecular dynamics simulation is dependent on the choice
+of integrator and the assumptions used in the integrator's derivation.
+For the commonly-used second order integrators such as the Verlet and
+Leapfrog algorithms, the velocities and accelerations should be
+approximately constant over the timestep. Thus, the timestep is limited
+by the highest frequency motion present in the system, which for
+all-atom simulations is usually bond vibrations. It is commonly found
+that using a timestep that is one tenth of this vibration's
+characteristic period is sufficient to conserve energy in the
+microcanonical ensemble. For example, if hydrogen molecules are present
+in the simulation box and the H-H bond vibration is the
+highest-frequency motion in the system, one can determine that with a
+force field harmonic force constant of approximately 500 N/m, the
+oscillation period will be 8 fs; thus, a 0.5 fs timestep can be used.
+For all-atom simulations with constraints on the high-frequency bonds,
+timesteps can be commonly increased to 2 fs; coarse-grained simulations
+with particles of higher mass and smaller force constants can have much
+larger timesteps.  After choosing a timestep, a test simulation should
+be run in the microcanonical ensemble to ensure that the choice of
+timestep yields dynamics that conserve energy.
```

Comment:
OK with also adding something like, "To some extent, this depends on the choice of integrator; some integrators [perhaps reference Chodera/Fass recent work] may allow larger timesteps."

And do you think we should mention hydrogen mass repartitioning?

Reply:
(I'm very glad you added this bit; it was a clear oversight on our part not to have it.)

Reply:
Definitely a good idea to add in a note that there are many ways to increase the timestep that can be used beyond that dictated by the smallest time period present in the system. Multiple-time-steps, hydrogen mass repartitioning, etc. can all be referenced as these kinds of methods.  I've added these.